cloudevents/languages/zh-CN/spec.md

```diff
@@ -270,22 +270,22 @@ CloudEvents 协议绑定或事件格式实现同样必须能够在规范字符
   - 必须是非空字符串
   - 应该以反向 DNS 名称作为前缀。该前缀域表明了定义此事件类型语义的组织。
 - 示例
-  - com.github.pull.create
-  - com.example.object.delete.v2
+  - com.github.pull_request.opened
```

Comment:
seems like we should make this change in the English version too for consistency... if we want to change it

Reply:
@duglin I checked the English version for these two lines, they've been changed already. 
https://github.com/cloudevents/spec/blob/676cef91678b86ae09fb39017588487690bfd42e/cloudevents/spec.md?plain=1#L350-L351

Reply:
cool thanks!